packages/modules/viewer-react/src/components/app-ui/providers/ViewerContentToolsProvider.tsx

```diff
@@ -0,0 +1,65 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import type { CommonStatusBarItem } from "@itwin/appui-abstract";
+import { StatusBarSection } from "@itwin/appui-abstract";
+import type { DefaultContentTools } from "@itwin/appui-react";
+import {
+  SectionsStatusField,
+  StandardContentToolsProvider,
+  StatusBarItemUtilities,
+  withStatusFieldProps,
+} from "@itwin/appui-react";
+import * as React from "react";
+
+export class ViewerContentToolsProvider extends StandardContentToolsProvider {
```

Comment:
I need to rethink subclassing BaseUiItemsProvider

Reply:
Originally I had `ViewerContentToolsProvider` as a Ui Provider, but the code was almost the exact same as `StandardContentToolsProvider` (with the exception of the `provideStatusBarItems` function). So I thought extending it and just overriding whatever was needed would result in the least amount of duplicated code.

Reply:
Yea for now this is fine, and i think probably what we'll end up doing.

The BaseUiItemsProvider class has an extra arg for chekcing frontstage support, and that opens up a can of worms...

But for now, this is fine, and lgtm